packages/keyring-api/src/api/account.ts

```diff
@@ -1,10 +1,8 @@
+import { object, UuidStruct } from '@metamask/keyring-utils';
```

Comment:
`FIXME:` we should move away from our `object` wrapper and push our fixes upstream.

Reply:
I've added a ticket to not forget about this:

- https://github.com/MetaMask/accounts-planning/issues/759